src/Tizen.NUI/src/public/Common/Color.cs

```diff
@@ -30,6 +30,11 @@ namespace Tizen.NUI
     [Tizen.NUI.Binding.TypeConverter(typeof(ColorTypeConverter))]
     public class Color : Disposable, ICloneable
```

Comment:
I think you have plan to remove 'Disposable' class inheritment in future.

Please write some TODO comments to `getCPtr`, `SwigCPtr`, and `CopyValueAndReturnSingletonHandleRef`.

BTW, Is it possible to remove inheritment of class? As I know it break ABI so it is impossible to change it.

Reply:
If we can't remove Disposable from Color, we can only don't create the SwigCPtr for Color.

Reply:
I check may scenarios, and looks find now.